model/clusters_mgmt/v1/azure_nodes_outbound_connectivity_types.model

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright (c) 2024 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// The configuration of the node outbound connectivity
+struct AzureNodesOutboundConnectivity {
+	// OutboundType is the type of network outbound configuration.
+	// The only accepted value is 'load_balancer'
```

Comment:
We should document that the default is `load_balancer`. Yes, `load_balancer` is the only accepted value (for now), but it should be clear that a default is set for it, the user doesn't explicitly need to set it. I'd also document that it is Immutable